addons/point_of_sale/views/res_partner_view.xml

```diff
@@ -15,6 +15,12 @@
                         <field string="POS Sales" name="pos_order_count" widget="statinfo"/>
                     </button>
                 </button>
+                <xpath expr="//group[@name='purchase']" position="after">
+                    <group string="Point Of Sale">
```

Comment:
and also adding `name` here to allow easy xpath 
( `<group string="Point Of Sale">`  --> `<group string="Point Of Sale" name="point_of_sale">` )

@switch87  What do you think on this ?

Reply:
Yes, you are right.

@pgu-odoo